### ISSUE TITLE
Assets Download Zip is Empty

### ISSUE THREAD

**Issue by MaketendoDev** [Jaysonjson/DMInternal#1]
![image](https://github.com/user-attachments/assets/df3d88bb-57fa-4470-8f87-24a522645c24)
is this supposed to be normal or did you just make it empty since the assets are from 1.21.3 and you'll release them once the update is released?

**Comment by Jaysonjson** [comment]
Yup, it's an empty file as DM 1.21.3 is still being worked on :)

**Comment by MaketendoDev** [comment]
ohhh alr. HOW THE FLIP DID YOU ZIP NOTHING!?!?!?! 💀

**Comment by Jaysonjson** [comment]
> ohhh alr. HOW THE FLIP DID YOU ZIP NOTHING!?!?!?! 💀

magic
